crates/sui-indexer/src/apis/indexer_api.rs

```diff
@@ -12,12 +12,14 @@ use jsonrpsee::{RpcModule, SubscriptionSink};
 
 use sui_core::event_handler::EventHandler;
 use sui_json_rpc::api::IndexerApiServer;
-use sui_json_rpc::api::{validate_limit, IndexerApiClient, QUERY_MAX_RESULT_LIMIT};
+use sui_json_rpc::api::{
```

Comment:
stacked PR, so maybe easier to review here https://github.com/MystenLabs/sui/pull/10275/commits/54d917a55c0448d106e5a5536ee558066a4c498d